packages/manager/apps/pci-public-ip/package.json

```diff
@@ -38,7 +38,8 @@
     "react-i18next": "^14.0.5",
     "react-router-dom": "^6.22.0",
     "react-use": "^17.5.0",
-    "zustand": "^4.5.0"
+    "zustand": "^4.5.0",
+    "lodash": "^4.17.15"
```

Comment:
**question:** Can we avoid using `lodash` since many of the functionalities are available in native JS, I dont see the need the need to add a library for this.